ArjunaJTS/jts/classes/com/arjuna/ats/jts/TransactionServer.java

```diff
@@ -109,6 +110,10 @@ static void doWork (String[] args, boolean exitOnComplete) throws Exception
 		else
 		    System.exit(0);
 	    }
+
+            if (args[i].compareTo("-recovery") == 0) {
+                RecoveryManager.manager().startRecoveryManagerThread();
```

Comment:
You should update the help command:
    System.out.println("Usage: [-otsname <name>] [-help] [-version]");